### PR TITLE
doc: nrf5340audio: edit legal section

### PR DESCRIPTION
**Pull request by greg-fer**
Added new legal contents requested by product managers.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

File: applications/nrf5340_audio/README.rst [hunk 1]
```diff
@@ -1268,6 +1270,10 @@ Legal notices for the nRF5340 Audio DK
 
       Nordic Semiconductor ASA customers using or selling these products for use in such applications do so at their own risk and agree to fully indemnify Nordic Semiconductor ASA for any damages resulting from such improper use or sale.
 
+   Radio frequency notice
+      The nRF5340 Audio development kit operates in the 2.4 GHz ISM radio frequency band.
+      The maximum radio-frequency power transmitted in the frequency band in which the development kit operates equals +3dBm (2 mW).
```
Comment (erikrobstad) [inline applications/nrf5340_audio/README.rst:1275]: no hyphen in "radio-frequency"?

File: applications/nrf5340_audio/README.rst [hunk 2]
```diff
@@ -1268,6 +1270,10 @@ Legal notices for the nRF5340 Audio DK
 
       Nordic Semiconductor ASA customers using or selling these products for use in such applications do so at their own risk and agree to fully indemnify Nordic Semiconductor ASA for any damages resulting from such improper use or sale.
 
+   Radio frequency notice
```
Comment (erikrobstad) [inline applications/nrf5340_audio/README.rst:1273]: The indentations (in the .rst file) looks inconsistent to me, but it looks fine in the built doc.

**Review by erikrobstad — APPROVED** [review]
LGTM as long as the text is approved by the right people.
Commented on some minor things.

File: applications/nrf5340_audio/README.rst (hunk 1 above)
Comment (greg-fer) [reply applications/nrf5340_audio/README.rst:1275]: It looked like a modifier to me, but on second thought, I think indeed it is not needed.

File: applications/nrf5340_audio/README.rst (hunk 2 above)
Comment (greg-fer) [reply applications/nrf5340_audio/README.rst:1273]: That's magic :D All is good formatting-wise.

**Review by FrancescoSer — APPROVED** [review]

**Review by PiChSwitz — APPROVED** [review]

**Pull request merged**